actions/flux/setup-flux-acr/action.yaml

```
@@ -30,3 +40,16 @@ runs:
+      id: get_acr_creds
+      shell: bash
+      env:
+        ACR_NAME: ${{ inputs.acr_name }}
+      run: |
+        credentials=$(az acr login --name ${ACR_NAME} --expose-token)
+        # Add log mask for accessToken and refreshToken to avoid leaking sensitive information using jq
+        accessToken=$(echo "${credentials}" | jq -r '.accessToken')
+        refreshToken=$(echo "${credentials}" | jq -r '.refreshToken')
+        echo "::add-mask::${accessToken}"
+        echo "::add-mask::${refreshToken}"
+        echo "credentials=${credentials}" >> $GITHUB_OUTPUT
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)